packages/cli/oclif.manifest.json

```diff
@@ -43,6 +43,17 @@
       "flags": {},
       "args": {}
     },
+    "auth:shop:create": {
```

Comment:
Auth is obviously not the right place for this. I just used it for the POC for simplicity.